hooks/command

```diff
@@ -131,29 +165,33 @@ done
 ## Releasing to heroku platform
 ##
 
-proc_type_names=()
-for line in $process_types; do
-  IFS=':' read -r -a tokens <<< "$line"
-  proc_type=${tokens[0]}
-  proc_type_names+=("$proc_type")
-done
-
-echo "+++ :heroku: Releasing ${proc_type_names[*]}"
+echo "+++ :heroku: Releasing ${releasing[*]}"
 
 updates=()
 
-for line in $process_types; do
+for line in "${process_type_images[@]}"; do
   IFS=':' read -r -a tokens <<< "$line"
   proc_type=${tokens[0]}
   proc_type_image=$(IFS=':'; echo "${tokens[*]:1}")
   proc_type_tag=$(get_proc_type_image_tag "$proc_type")
+
   # Based on https://devcenter.heroku.com/articles/container-registry-and-runtime#getting-a-docker-image-id
   image_id=$(docker inspect "$proc_type_tag" --format=\{\{.Id\}\})
 
-  echo "Inspected $proc_type_tag identified as $image_id"
-  updates+=("{\"type\":\"${proc_type}\",\"docker_image\":\"$image_id\"}")
+  # shellcheck disable=SC2076
+  if [[ " ${releasing[*]} " =~ " ${proc_type} " ]]; then
```

Comment:
is this making sure the release target exist in the "process type images"?

Reply:
This is filter all the process type images with the `releasing` array. so that only process type images included in the `releasing` array are to be released to the heroku formation api.